lib/Dialect/TritonGPU/IR/Ops.cpp

```diff
@@ -671,10 +672,6 @@ LogicalResult MemDescSubviewOp::verify() {
         return emitError("only constant values are allowed outside the front "
                          "dimension in a rank-reducing subview");
       }
-      if (!value.isZero()) {
```

Comment:
I don't think this is correct without changing the lowering. Is this tested anywhere?

Reply:
This repro does not exercise this path. See that this path is for rank-reducing views, like those used in pipelining